Add ExternalIssuers, refactor statistics APIs (#1198)
google/exposure-notifications-verification-server: docs/api.md

  the padding.
* `uuid` is optional as request input. The server will generate a uuid on response if omitted.
  * This is a handle which allows the issuer to track status of the issued verification code.

> * `externalIssuerID` is a optional information supplied by the API caller to

Reviewer: ```suggestion
* `externalIssuerID` is an optional field supplied by the API caller to
```

Author: Done
